html/Good_practices_in_Web_Dev/good-practices-in-web-dev.md

```diff
@@ -0,0 +1,248 @@
+# Good practices in Web-Dev
+
+To make sure quality user experience, your website must be displayed correctly across all the devices. To achieve this, we need to write scalable code and give the same output for all the devices.
+
+## what will it change?
```

Comment:
Start with the uppercase letter: "What will it change". 
A better title will be "How this is going to help you"